app/views/home/dashboard.html.erb

```diff
@@ -1,144 +1,149 @@
 <div class="col-md-8 no-padding">
-  <div class="panel panel-default module">
+  <div class="panel panel-default module module-videos">
     <div class="panel-heading">
       <b>In Progress Videos</b>
-      <a class="pull-right" href="#"><i class="fa fa-plus-circle"></i></a>
+      <a class="pull-right" href="/videos"><i class="fa fa-plus-circle"></i></a>
```

Comment:
Wrong path. Again, don't use absolute paths. Rails gives nice URL helpers for these.